_proto/hapi/chart/metadata.proto

```diff
@@ -17,6 +17,10 @@ message Maintainer {
 //
 // 	Spec: https://k8s.io/helm/blob/master/docs/design/chart_format.md#the-chart-file
 message Metadata {
+	enum Engine {
+		// gotpl is the default engine
+		gotpl = 0;
```

Comment:
style: change 'gotpl' to 'GOTPL' (proto uses capitals with underscores for enum values.). Also, using 0 to represent the default could be dangerous, because unsupported values will default to GOTPL, i.e. making it hard to distinguish between someone supplying a default vs. supplying an incorrect value.